OpenRA.Game/Game.cs

```diff
@@ -49,6 +49,8 @@ public static class Game
 
 		public static MersenneTwister CosmeticRandom = new MersenneTwister(); // not synced
 
+		public static FontManager FontManager { get; private set; }
```

Comment:
Fonts are all mod-defined data so ideally this should be a field on the `ModData`. This might slightly complicate the initialization (getting the window to its ctor) but should remove the larger complications around static state, `InitializeFonts`, and disposal.

We could keep this as `public static FontManager FontManager => ModData.FontManager;` if we have users that don't have direct access to the `ModData` instance.

Reply:
If we want to move FontManager to ModData, then we would need to store `IPlatformWindow` on the Game class and inject it in the constructor on the ModData class.  IMO it is as ugly as we have now,

Reply:
I wouldn't say its clean, but it is definitely less ugly than we have now because the platform window has the same lifetime/scope as Game (created once at startup and is only disposed when the game is closed), while fonts are disposed and recreated each time we switch mods.

The fonts getting out of sync with ModData during mod switch was the cause of #17840 and probably other issues.

Reply:
Agree this suggestion seems nicer on balance.